server/Arcadia.Assistant.Web/Authorization/Requirements/UserIsHealthRequirement.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Arcadia.Assistant.Web.Authorization.Requirements
+{
+    using Microsoft.AspNetCore.Authorization;
+
+    public class UserIsHealthRequirement:IAuthorizationRequirement
```

Comment:
no need for that

Reply:
Why? It is needed to distinguish authorization handlers.